src/commands/help.ts

```diff
@@ -0,0 +1,33 @@
+import { Command } from "../lib/Command";
+import { CommandArg, CommandExecuteParameters } from "../types";
+import bot from "../app";
+
+const options: CommandArg[] = []
+const cooldown = 5;
+
+const playerCount = async (options: CommandExecuteParameters) => {
+  const { type, message, interaction } = options
+  const isSlash = type === "interaction";
+  var list = ["class", "github", "help", "matchmaking", "playercount", "schedule", "skill", "stats", "whohas"];
```

Comment:
You can use the bot.commands object, instead of making the list yourself.
This way you don't have to update this file when you make a new command